config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -41,26 +41,23 @@ presubmits:
     cluster: k8s-infra-prow-build
     skip_branches:
     - release-\d+.\d+ # per-release job
-    labels:
-      preset-service-account: "true"
```

Comment:
this should only be necessary for bootstrap jobs
this job is decorated (and the non-experimental) we should be able to drop this from both but starting here

this is relevant here because this + runner.sh cause us to try to activate the credentials / use gcloud, which fails (but we shouldn't need to use gcloud, so just stop using it instead of trying to make that available to non-root)